pkg/util/image.go

```diff
@@ -49,10 +51,43 @@ func GetBackingImageDataSourceName(backingImageCache ctllhv1.BackingImageCache,
 	return GetBackingImageName(backingImageCache, image)
 }
 
-func GetImageStorageClassName(imageName string) string {
+func storageClassLegacyName(imageName string) string {
 	return fmt.Sprintf("longhorn-%s", imageName)
 }
 
+func GenerateStorageClassName(imageNamespace, imageUID string) string {
+	return lhutil.AutoCorrectName(fmt.Sprintf("longhorn-%s-%s", imageNamespace, imageUID), lhdatastore.NameMaximumLength)
+}
```

Comment:
```suggestion
func GenerateStorageClassName(imageNamespace, imageUID string) string {
	return lhutil.AutoCorrectName(fmt.Sprintf("longhorn-%s", imageUID), lhdatastore.NameMaximumLength)
}
```
As we plan to use image uid to name sc, maybe namespace is not necessary?

Reply:
Thank you. I update to use UID only.